src/urdf_parser_py/xml_reflection/core.py

```diff
@@ -580,7 +580,7 @@ def to_xml(self):
         """ Creates an overarching tag and adds its contents to the node """
         tag = self.XML_REFL.tag
         assert tag is not None, "Must define 'tag' in reflection to use this function"  # noqa
-        doc = etree.Element(tag)
+        doc = ET.Element(tag)
```

Comment:
I guess we are inheriting `ET` by importing from `basics.*` at the top, but I think it would be a bit better to explicitly do it at the top of this file:

```
from xml.etree import ElementTree as ET
```

Minor comment, though, I'll leave it to you whether you think we should do that.

Reply:
I removed the `*` import from `core.py` in 0ae72598fda6386db175e4e5d6562522de244541 . `sdf` and `urdf` also `import *`, but I'd like to fix that in a follow up PR.